{"name": "test_estimate_bridge_native_to_not_native[Polygon (MATIC)]", "status": "failed", "statusDetails": {"message": "AssertionError: Ожидался status = 200, пришел status = 400. Тело ответа: \n{\"ok\":false,\"data\":null,\"error\":\"Request failed with status code 400\"}.", "trace": "network = 'Polygon (MATIC)'\n\n    @pytest.mark.parametrize(\"network\", networks)\n    def test_estimate_bridge_native_to_not_native(network):\n        net = de_bridge_keys_data()[network].get('net')\n        native_token = de_bridge_keys_data()[network].get('nativeToken')\n        not_native_token = de_bridge_keys_data()[network].get('notNativeToken')\n>       DeBridge().EstimateBridge().estimate_bridge_native_to_not_native(net=net, native_token=native_token,\n                                                                         not_native_token=not_native_token)\n\ntests/api_tests/tests_de_bridge.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <serviceCalls.debrdige_service.DeBridge.EstimateBridge object at 0x10ea61a00>\nnet = 'polygon', native_token = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee'\nnot_native_token = '0xa1428174f516f527fafdd146b883bb4428682737'\n\n    def estimate_bridge_native_to_not_native(self, net, native_token, not_native_token):\n        response = requests.get(\n            f\"{self.url}?srcNet={net}&srcTokenAddress={native_token}\"\n            f\"&srcTokenAmount=1&dstNet={net}&dstTokenAddress={not_native_token}\")\n        logger.info(f\"Отправлен запрос на роут - {response.url}\")\n        logger.info(response.text)\n>       assert response.status_code == 200, f\"Ожидался status = {200}, \" \\\n                                            f\"пришел status = {response.status_code}. \" \\\n                                            f\"Тело ответа: \\n{response.text}.\"\nE       AssertionError: Ожидался status = 200, пришел status = 400. Тело ответа: \nE       {\"ok\":false,\"data\":null,\"error\":\"Request failed with status code 400\"}.\n\nserviceCalls/debrdige_service.py:39: AssertionError"}, "attachments": [{"name": "log", "source": "d66cd32c-dfb4-49d9-9e88-856a365f2ec4-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "network", "value": "'Polygon (MATIC)'"}], "start": 1685610392714, "stop": 1685610393353, "uuid": "77ba7503-6060-4bf8-9582-d89ae623e9b1", "historyId": "51a8aaebb40bb68f46ba328a29561e49", "testCaseId": "02aafaeb5599f02847fcd6b690f3a745", "fullName": "tests.api_tests.tests_de_bridge#test_estimate_bridge_native_to_not_native", "labels": [{"name": "parentSuite", "value": "tests.api_tests"}, {"name": "suite", "value": "tests_de_bridge"}, {"name": "host", "value": "MacBook-Air-Anna.local"}, {"name": "thread", "value": "16727-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_tests.tests_de_bridge"}]}